packages/mdc-checkbox/foundation.ts

```diff
@@ -48,12 +48,14 @@ export class MDCCheckboxFoundation extends MDCFoundation<MDCCheckboxAdapter> {
       isIndeterminate: () => false,
       removeClass: () => undefined,
       removeNativeControlAttr: () => undefined,
+      setChecked: () => undefined,
       setNativeControlAttr: () => undefined,
       setNativeControlDisabled: () => undefined,
     };
   }
 
   private currentCheckState_ = strings.TRANSITION_STATE_INIT;
+  private realCheckState_ = false;
```

Comment:
Can you add a comment above this line that provides more info, something like:
```
// Native checkboxes can only have two real states: checked/true or unchecked/false
// The indeterminate state is visual only.
```

Reply:
And add link to the stackoverflow question that explains this too: https://stackoverflow.com/a/33529024